site/constants/index.ts

```diff
@@ -336,3 +336,7 @@ export const NEXTDOOR_AUTH_URL = "https://auth.nextdoor.com/";
 export const NEXTDOOR_URL = "https://nextdoor.co.uk/";
 
 export const MAX_OPERATOR_NOC_CODES = 5;
+
+export const NETWORK_AREA_FEATURE_FLAG = !["preprod", "prod"].includes(STAGE);
+
+export const exclusions = ["110", "143", "146", "147"];
```

Comment:
Might be nice to give this a more descriptive name :) 

```suggestion
export const NETWORK_CONSEQUENCE_ADMIN_AREA_EXCLUSIONS = ["110", "143", "146", "147"];
```

Reply:
done